<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowobj_fname_browser</class>
 <widget class="QMainWindow" name="MainWindowobj_fname_browser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Sleepace Data View</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QScrollArea" name="obj_fname_list">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>80</y>
      <width>491</width>
      <height>361</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="obj_fname_widget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>489</width>
       <height>359</height>
      </rect>
     </property>
     <widget class="QTextBrowser" name="obj_fname_browser">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>491</width>
        <height>361</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label_dev">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>DEVICE</string>
    </property>
   </widget>
   <widget class="QComboBox" name="obj_dev_list">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>104</width>
      <height>26</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>LZ300</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>P300</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>M500/M600</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Binatone</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="obj_mode_list">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>40</y>
      <width>104</width>
      <height>26</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>server</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>app</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_mode">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>20</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>MODE</string>
    </property>
   </widget>
   <widget class="QComboBox" name="obj_alg_list">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>40</y>
      <width>104</width>
      <height>26</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>binatone-v1.0</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Zx-1.1.1-9</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_alg">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>20</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>ALGORITHM</string>
    </property>
   </widget>
   <widget class="QComboBox" name="obj_fs_list">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>40</y>
      <width>104</width>
      <height>26</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>100Hz</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>50Hz</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_fs">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>20</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>SAMPLING RATE</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="obj_detail">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>460</y>
      <width>491</width>
      <height>90</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="obj_button_plot">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>507</y>
      <width>200</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Plotting</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="obj_input_fname">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>80</y>
      <width>200</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>140</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string extracomment="原始信号和增益（编码、解码）">Raw Data (gain)</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_2">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>180</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Offbed Data</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_3">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>220</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Heart Rate Data</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_4">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>260</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Resp Rate Data</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_5">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>300</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Sleep Cycle Analysis</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>440</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>&gt;&gt; detail</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_6">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>340</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>All Data (not SCA)</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_7">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>380</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Report Data</string>
    </property>
   </widget>
   <zorder>obj_button_plot</zorder>
   <zorder>obj_dev_list</zorder>
   <zorder>obj_fname_list</zorder>
   <zorder>label_dev</zorder>
   <zorder>obj_mode_list</zorder>
   <zorder>label_mode</zorder>
   <zorder>obj_alg_list</zorder>
   <zorder>label_alg</zorder>
   <zorder>obj_fs_list</zorder>
   <zorder>label_fs</zorder>
   <zorder>obj_detail</zorder>
   <zorder>obj_input_fname</zorder>
   <zorder>checkBox</zorder>
   <zorder>checkBox_2</zorder>
   <zorder>checkBox_3</zorder>
   <zorder>checkBox_4</zorder>
   <zorder>checkBox_5</zorder>
   <zorder>label</zorder>
   <zorder>checkBox_6</zorder>
   <zorder>checkBox_7</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
